packages/node_modules/@ciscospark/widget-space/src/selector.js

```diff
@@ -5,23 +5,36 @@ const getWidget = (state) => state.widgetSpace;
 const getSpark = (state) => state.spark;
 const getMedia = (state) => state.media;
 const getConversation = (state) => state.conversation;
-const getUsers = (state) => state.users;
 const getCurrentUser = (state, ownProps) => ownProps.currentUser;
 const getFeatures = (state) => state.features;
 const getOwnProps = (state, ownProps) => ownProps;
 
+/**
+ * Get the other user in a 1:1 conversation.
+ * @param {conversation} conversation
+ * @param {participant} currentUser
+ * @returns {participant}
+ */
+export const getToUser = createSelector(
+  [getConversation, getCurrentUser],
+  /* eslint arrow-body-style: [0] */
+  (conversation, currentUser) => {
+    return conversation
+      .get('participants')
+      .filter((participant) => participant.get('id') !== currentUser.id)
```

Comment:
`.find` would do the same thing as `.filter().first()`

Reply:
I like the efficiency. I will update.